config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -89,4 +88,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
+  replaces: rhpam-kogito-operator.v7.12.0-1
```

Comment:
Replace version not exists, so this csv will fail build of indexing image. There should be used `7.12.0-1` see https://github.com/kiegroup/rhpam-kogito-operator/blob/7.12.0.GA/bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml#L421-L422

Reply:
without v?

Reply:
without this part `rhpam-kogito-operator.v` as previous version is 7.12.0-1